### ISSUE TITLE
getFeed / magpie - Support for latest WordPress (3.0.1) datafeed?

### ISSUE THREAD

**Issue by pixelchutes** [splittingred/getFeed#6]
Has anyone been able to get getFeed to correctly parse a blog feed from a clean install of the latest WordPress (v.3.0.1)? getFeed is functioning fine for other sample feeds, but I am running into magpie parse errors related to the WordPress feed?

```
(ERROR @ /modx/revolution/core/model/modx/xmlrss/rssfetch.class.php : 242) User warning: MagpieRSS: Failed to fetch http://localhost/modx/revolution/blog/?feed=rss2 (HTTP Response: HTTP/1.0 301 Moved Permanently)
```

NOTE: This is the same error whether I have "FURLs" enabled for WordPress or not...

**Comment by austinpickett** [comment]
Not too sure what MagPie is, but i'm having the same issue as it won't take my 
http://blog.com/blog/feed/

**Comment by pixelchutes** [comment]
FYI, I was ultimately able to get it to work by manually updating the core "snoopy.class.php" with MODx Revolution. (This is generally not recommended as it will not be preserved upon future upgrades, etc)

However, it is a solution nonetheless :)

http://modxcms.com/forums/index.php/topic,51232.msg307230.html#msg307230

**Comment by pixelchutes** [comment]
I have filed a Bug pertaining to this issue: http://bugs.modx.com/issues/2589

It does not seem to be related to getFeed directly, rather core Snoopy file shipped w/ Revo.

**Comment by pixelchutes** [comment]
Sweet! Thx Shaun. Happy to see this will be released in Revolution-2.0.5-pl :)
